ui/basicTab/basicTab.pug

```diff
@@ -2,7 +2,7 @@ include ../global/advantages.pug
 
 // Basic section 
 .basic.section.basic-tab
-  .advantages-and-feats.horizontal-flex
+  .advantages-and-feats
```

Comment:
As a project grows larger, it's generally better to name classes by concept, not by style, and style them specifically.